### ISSUE TITLE
Persist local-repo setting for history/restore ops

### ISSUE THREAD

**Issue by spyrkob** [spyrkob/prospero#45]
restore, history and update ops should reuse the same local-repo settings as install

**Comment by TomasHofman** [comment]
Can we extend prospero config to keep local-repo path?

**Comment by spyrkob** [comment]
Yes, I was thinking of renaming `repositories` to `remote-repositories` and adding `local-repository` element. `local-repository` would then only take URL
